server/src/main/java/org/elasticsearch/common/blobstore/OperationPurpose.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.common.blobstore;
+
+public enum OperationPurpose {
+    SNAPSHOT("Snapshot"),
+    CLUSTER_STATE("ClusterState"),
+    INDICES("Indices"),
+    TRANSLOG("Translog");
+
+    private final String value;
+
+    OperationPurpose(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
```

Comment:
I wonder about this name. I'd call it `getStatsKey` or just `getKey` rather than anything with `value`. Perhaps you have a specific motivation for the name though?

Reply:
No specific reason. The word `value` is my default choice when I don't have a good name. I updated it to `getKey`.